instagram-app/src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import dummyData from './dummy-data';
+import SearchBar from './components/SearchBar/SearchBar.js';
+import PostContainer from './components/PostContainer/PostContainer.js'
+import './App.scss';
+
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      data: dummyData
+    };
+  }
+  render() {
+    return (
+      <div>
+        <SearchBar />
+        <PostContainer key={this.state.data.id} postData={this.state.data} />
+      </div>
+    )
```

Comment:
Interesting you are passing a key into your props object here, is there any reason?